fields.php

```diff
@@ -34,6 +36,23 @@ function voicewp_fm_alexa_app_settings() {
 		) ),
 	);
 
+	// If there's a post ID, output the REST endpoint for use in the amazon developer portal
+	if ( $post_id ) {
+		$children['readonly_skill_url'] = new \Fieldmanager_TextField( array(
+			'label' => __( 'This is the endpoint URL of your skill. Paste this within the configuration tab for your skill in the developer portal.', 'voicewp' ),
+			'default_value' => home_url( '/wp-json/voicewp/v1/skill/' ) . $post_id,
```

Comment:
This is missing a trailing slash.  Also, would it make more sense to concatenate first?